kuma/wiki/jinja2/wiki/document.html

```diff
@@ -262,14 +262,24 @@
 
               {% if document.current_revision %}
                 {% if document.current_revision.needs_technical_review %}
-                  <div class="warning warning-review"><p>{% trans %}
-                    This article is in need of a technical review.
-                  {% endtrans %}</p></div>
+                  <div class="warning warning-review">
+                    <p>
+                      {% trans %}This article needs a technical review.{% endtrans %}
+                      <a href="https://developer.mozilla.org/docs/MDN/Contribute/Howto/Do_a_technical_review">
+                        {% trans %}How you can help.{% endtrans %}
+                      </a>
+                    </p>
```

Comment:
Jinja2 trans tag allows for variables, so you could do this like so:

```
{% trans url="https://developer.mozilla.org/docs/MDN/Contribute/HowTo/Do_a_technical_review" %}
  This article needs a technical review. <a href="{{ url }}">How you can help.</a>
{% endtrans %}
```

Ditto for the other text changes.